unionvms-web/app/partial/header/headerMenu/headerMenu.js

```diff
@@ -26,6 +26,17 @@ angular.module('unionvmsWeb').controller('HeaderMenuCtrl',function($scope, $root
         return userService.isAllowed(feature, module, true);
     };
 
+     var checkReviewContactDetails = function () {
+        if (userService.isLoggedIn()) {
+            changesService.isReviewContactDetailsEnabled().then(
+                function (response) {
+                     console.log(response);
+                     return response.isReviewContactDetailsEnabled;
+                });
+         }
+         return false;
```

Comment:
hello, I'm not sure this will work. The function will most probably reach line 37 and return false before the request is answered so that the code inside the 'then' function is executed.